phpcs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<ruleset name="Budget">
+  <rule ref="Generic.Arrays.DisallowLongArraySyntax" />
```

Comment:
Extend with more rules (for example with PSR1)

Reply:
In due time, let's start